gk/main.c

```diff
@@ -49,6 +51,9 @@
 /* Set @START_ALLOWANCE as the double size of a large DNS reply. */
 #define	START_ALLOWANCE		 (8)
 
+/* The batch size for IPv4 LPM table lookup. */
+#define	FWDSTEP                  (4)
```

Comment:
`FWDSTEP` is only used inside of `lookup_fib_bulk()` and `lookup_fib_bulk()` doesn't compensate if someone changes `FWDSTEP` to another value, nor have an `RTE_BUILD_BUG_ON(FWDSTEP != 4)`. I think it's better to just define `FWDSTEP` as a const inside of `lookup_fib_bulk()`.